doc/onboarding.md

```diff
@@ -0,0 +1,88 @@
+# Numberscope onboarding
+
+## Accounts, etc.
+
+1. If you don't have an account, sign up for [GitHub](https://github.com).
+2. Make sure you've been added to the Numberscope GitHub organization.
+3. Make sure you've been added to the chat app (Slack or Zulip).
+
```

Comment:
Since this page will be available to anyone visiting the public instance of Numberscope, should you mention that (2) and (3) apply only to members of the UC Boulder Experimental Math Lab?

Reply:
Added note in https://github.com/numberscope/frontscope/pull/148/commits/59fd25623bc1f78bae3e0e097eb8fbecb53e6926.